signatures/signature.go

```diff
@@ -67,26 +67,28 @@ func init() {
 
 var nullAddr = common.BytesToAddress([]byte{})
 
-func Address(o *signatures.Ownership) (common.Address, error) {
-	// in the case of conditions, all signatures are treated similarly to produce an address
-	// and we just take the hash of the public key and the conditions and produce an address
-	if o.Conditions != "" {
-		pubKeyWithConditions := append(o.PublicKey.PublicKey, []byte(o.Conditions)...)
```

Comment:
Want to specifically call out this change as it is breaking... we never exposed conditions so it's ok, but this changes how conditions were calculated from using the public key to instead using an Address which makes it so address can be used for ownership without knowing the public key behind them.